src/binding/python/BaseRecordComponent.cpp

```diff
@@ -53,17 +53,17 @@ void init_BaseRecordComponent(py::module &m) {
                 return py::dtype("b");
             else if( brc.getDatatype() == DT::UCHAR )
                 return py::dtype("B");
-            else if( brc.getDatatype() == DT::INT16 )
+            else if( brc.getDatatype() == DT::SHORT )
                 return py::dtype("int16");
-            else if( brc.getDatatype() == DT::INT32 )
+            else if( brc.getDatatype() == DT::INT )
                 return py::dtype("int32");
-            else if( brc.getDatatype() == DT::INT64 )
+            else if( brc.getDatatype() == DT::LONG )
                 return py::dtype("int64");
-            else if( brc.getDatatype() == DT::UINT16 )
+            else if( brc.getDatatype() == DT::USHORT )
                 return py::dtype("uint16");
-            else if( brc.getDatatype() == DT::UINT32 )
+            else if( brc.getDatatype() == DT::UINT )
                 return py::dtype("uint32");
-            else if( brc.getDatatype() == DT::UINT64 )
+            else if( brc.getDatatype() == DT::ULONG )
```

Comment:
This is now probably not precise enough as `ULONG` may be 32 or 64.
Numpy provides a number of compatible C types, but not all of the applicable ones:
https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.scalars.html#built-in-scalar-types

Reply:
I see you recorgnized the problem in `RecordComponent.cpp` already. :+1:

Reply:
Yep, pls feel free to annotate any line you catch :)